a3p-integration/proposals/z:acceptance/test-lib/psm-lib.js

```diff
@@ -22,14 +23,20 @@ import {
   mkTemp,
   VALIDATORADDR,
 } from '@agoric/synthetic-chain';
+import { spawn } from 'node:child_process';
 import fsp from 'node:fs/promises';
+import { Readable } from 'node:stream';
 import { NonNullish } from './errors.js';
 import { getBalances } from './utils.js';
 
+/** @import {CommandResult} from '@agoric/internal'; */
+
 // Export these from synthetic-chain?
 const USDC_DENOM = NonNullish(process.env.USDC_DENOM);
 const PSM_PAIR = NonNullish(process.env.PSM_PAIR).replace('.', '-');
 
+const runCommand = makeRunCommand({ Buffer, Readable, setImmediate, spawn });
```

Comment:
I'm reluctant to make a new API for what Execa offers. Can you make the case for this? (Or convert to use Execa?)

Execa has ambient authority, but I think that's acceptable in testing libs. Pretty sure that's in our guidelines.

Reply:
Yeah, it looks like execa can handle this. I'll see about refactoring on top of it after CI passes with this.